suite-common/wallet-config/src/networksConfig.ts

```diff
@@ -1,6 +1,50 @@
 import { DeviceModelInternal } from '@trezor/connect';
 
-import { Networks } from './types';
+import { Networks, NetworkType, Explorer } from './types';
+
+export const getExplorerUrls = (
+    baseUrl: string,
+    networkType: NetworkType,
+    solanaDevnet?: boolean,
+): Explorer => {
+    switch (networkType) {
+        case 'bitcoin':
+            return {
+                tx: `${baseUrl}/tx/`,
+                account: `${baseUrl}/xpub/`,
+                address: `${baseUrl}/address/`,
+            };
+        case 'ethereum':
+            return {
+                tx: `${baseUrl}/tx/`,
+                account: `${baseUrl}/address/`,
+                address: `${baseUrl}/address/`,
+                nft: `${baseUrl}/nft/`,
+            };
+        case 'ripple':
+            return {
+                tx: `${baseUrl}/tx/`,
+                account: `${baseUrl}/account/`,
+                address: `${baseUrl}/account/`,
+            };
+        case 'solana':
+            return {
+                tx: `${baseUrl}/tx/`,
+                account: `${baseUrl}/account/`,
+                address: `${baseUrl}/account/`,
+                queryString: solanaDevnet ? `?cluster=devnet` : '',
+            };
+        case 'cardano':
+            return {
+                tx: `${baseUrl}/tx/`,
+                account: `${baseUrl}/address/`,
+                address: `${baseUrl}/address/`,
+                token: `${baseUrl}/asset/`,
+            };
+        default:
+            throw new Error(`Unsupported network type: ${networkType}`);
```

Comment:
I see I'm late to the party already :smile:  Very nice refactoring :+1: 
Great to see exhaustive typing, which is very helpful pattern :heart: 
But I'd suggest a tiny improvement to this. I'll make a PR, it will be quicker than to describe it..